interop-testing/build.gradle

```diff
@@ -69,10 +69,18 @@ task reconnect_test_client(type: CreateStartScripts) {
     classpath = jar.outputs.files + configurations.runtime + configurations.tcnative
 }
 
+task stresstest_client(type: CreateStartScripts) {
+    mainClassName = "io.grpc.testing.integration.StressTestClient"
+    applicationName = "stresstest-client"
```

Comment:
Should consider what diagnostic logging from them VM would be useful when running the client. Vebose GC is probably pretty important, not sure if we want a gauge for system memory etc. Can add later